templates/assets/stylesheets/frontend/layout/body.sass

```diff
@@ -0,0 +1,2 @@
+body
+  font-family: $font-family-base, sans-serif
```

Comment:
`sans-serif` už je obsažený v proměnné `$font-family-base`

je to vůbec potřeba? když už je to samé definované v Bootstrapu? Nebo to je pro případ, že začínáš projekt bez Bootstrapu?